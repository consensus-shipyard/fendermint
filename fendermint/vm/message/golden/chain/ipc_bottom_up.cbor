a163497063a168426f74746f6d5570a2676d657373616765a3676d657373616765a26a636865636b706f696e74a4697375626e65745f6964821bffffffffffffffff8356040a150a15a7f73119c5fff59f4536fdff1e6628f8924b00bd8ac9ece6a196ae8c014b0088ee8bacc9ab83cbea01666865696768741a61f7b23f756e6578745f76616c696461746f725f7365745f69641bab4526659dc9c1d272626f74746f6d5f75705f6d65737361676573d82a5823001220d516016a37df05d05b0bc1d0859e6ccaef3a39f98cf83122def22df12a517fa06a7369676e61747572657383a26976616c696461746f725502add901ff4cbd310000b279519409aede7b206bd2697369676e61747572654101a26976616c696461746f724a00c790fb9dddacdfe92f697369676e61747572654102a26976616c696461746f725004d8df80b1a5fafdf5fa0100464f5614697369676e61747572654701bb3bb728ed506772656c617965725831032e4b673d0e6f8bcfb6015952eabe29e23a05bbba00f63e5ee03f91ffa5484868b794740fd2e7d3a21a000197ffd0b6006873657175656e63651b9345babad4844692697369676e61747572654301ffa3