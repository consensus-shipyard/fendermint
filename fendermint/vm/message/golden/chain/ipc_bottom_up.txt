Ipc(BottomUp(SignedRelayedMessage { message: RelayedMessage { message: SignedBottomUpCheckpoint { checkpoint: BottomUpCheckpoint { subnet_id: SubnetID { root: 18446744073709551615, children: [Address { payload: Delegated(DelegatedAddress { namespace: 10, length: 20, buffer: [21, 10, 21, 167, 247, 49, 25, 197, 255, 245, 159, 69, 54, 253, 255, 30, 102, 40, 248, 146, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0] }) }, Address { payload: ID(10114056781670008125) }, Address { payload: ID(16903712942678275848) }] }, height: 1643622975, next_validator_set_id: 12341312571759509970, bottom_up_messages: Cid(QmcgT1b8CJhRtmqFsnX2QVUmiMYLFwcViQGVbAi9UWfyRq) }, signatures: [ValidatorSignature { validator: Address { payload: Actor([173, 217, 1, 255, 76, 189, 49, 0, 0, 178, 121, 81, 148, 9, 174, 222, 123, 32, 107, 210]) }, signature: Signature { sig_type: Secp256k1, bytes: [] } }, ValidatorSignature { validator: Address { payload: ID(3446236016166225991) }, signature: Signature { sig_type: BLS, bytes: [] } }, ValidatorSignature { validator: Address { payload: Delegated(DelegatedAddress { namespace: 18080817611769130968, length: 5, buffer: [0, 70, 79, 86, 20, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0] }) }, signature: Signature { sig_type: Secp256k1, bytes: [187, 59, 183, 40, 237, 80] } }] }, relayer: Address { payload: BLS([46, 75, 103, 61, 14, 111, 139, 207, 182, 1, 89, 82, 234, 190, 41, 226, 58, 5, 187, 186, 0, 246, 62, 94, 224, 63, 145, 255, 165, 72, 72, 104, 183, 148, 116, 15, 210, 231, 211, 162, 26, 0, 1, 151, 255, 208, 182, 0]) }, sequence: 10612093408560563858 }, signature: Signature { sig_type: Secp256k1, bytes: [255, 163] } }))